examples/pkg/clusterclient/typed/example/v1/examplev1.go

```diff
@@ -78,76 +77,76 @@ func (w *wrappedClusterTestType) checkCluster(ctx context.Context) (context.Cont
 	return ctx, nil
 }
 
-// Create implements ClusterTestTypeInterface.
```

Comment:
The generated methods aren't sorted but in the order as specified [here](https://github.com/kcp-dev/code-generator/blob/d5edbfdfa54f819b08c5285b1ffe1e7c9bc6bf54/pkg/internal/parser.go#L244-L321) which is the order that code-gen [follows](https://github.com/kubernetes/code-generator/blob/7d977b3e5454e47ec6b903242616e9ea62644226/cmd/client-gen/generators/fake/generator_fake_for_type.go#L190-L215). And previously the methods were hardcoded in templates instead of being split as done here. Really sorry for the clumsy diff :(